packages/sdk/shell/src/panels/IdentityPanel/steps/IdentityActionChooser.tsx

```diff
@@ -54,22 +47,30 @@ export const IdentityActionChooserImpl = ({
   onCreateInvitationClick,
   devices,
   connectionState,
-  onAgentDestroy,
-  onAgentCreate,
-  agentStatus,
-  agentActive,
   agentHostingEnabled,
+  ...agentProps
 }: IdentityActionChooserImplProps) => {
+  const { nonAgentDevices, agentDevice } = devices.reduce(
```

Comment:
worth noting that there could still be other agents in here but likely no managed one so this maybe would be better name "nonManagedDevices" and "managedDevice"